app/scripts/metamask-controller.js

```diff
@@ -3191,6 +3196,7 @@ export default class MetamaskController extends EventEmitter {
         txController.updatePreviousGasParams.bind(txController),
       abortTransactionSigning:
         txController.abortTransactionSigning.bind(txController),
+      getLayer1GasFee: txController.getLayer1GasFee.bind(txController),
```

Comment:
Since we have so many properties here, we'll have to go alphabetical eventually... 😄